### PR TITLE
DCS-1029 implemented new support page

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

**Review by JamieDraperUK — APPROVED** [review]
LGTM

File: server/views/partials/footer.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<footer class="govuk-footer " role="contentinfo">
```
Comment (andrewrlee) [inline server/views/partials/footer.html:1]: is there a reason why we cant use the govuk footer macro with this?

Comment (louisenorris) [reply server/views/partials/footer.html]: Was initially following the html format of header and layout already in place but seems like no reason not to use the nunjucks macro - implemented as suggested

File: server/views/help-reporting-use-of-force.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            
+        <h2 class="govuk-heading-m govuk-!-padding-10">Technical support</h2>
+          <p>For technical support, you should contact the helpdesk by calling {{supportTelephone}}, or {{supportExtension}} from inside an establishment.</p>    
+      </div>
+    </div>
+  </div>
+{% endblock %}
+
+{% block footer %}
+{{  govukFooter({}) }}
```
Comment (andrewrlee) [inline server/views/help-reporting-use-of-force.html:37]: meh... blocks/if/for etc should increase indentation

**Review by andrewrlee — APPROVED** [review]
LGTM

File: server/views/help-reporting-use-of-force.html (hunk 2 above)
Comment (louisenorris) [reply server/views/help-reporting-use-of-force.html:37]: Ah - of course sorry

**Pull request merged**